plugin-management-library/pom.xml

```diff
@@ -50,6 +50,11 @@
             <version>1.19.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
```

Comment:
It will make it more difficullt to consume it from the Jenkins core, but I have a plan for it (forked process)